WorkData/PCEDT/data/wsj0001.cz.m

```diff
@@ -12,221 +12,161 @@
 <LM id="manual">
 <desc>Manual annotation</desc>
 </LM>
-<LM id="tagger-a"></LM>
+<LM id="tagger-a"/>
```

Comment:
Did you run a script in btred? Just curious, because things like these are happening constantly:
`<schema />` turning into `<schema/>`
`<LM></LM>` turning into `<LM/>`